### DISCUSSION TITLE
negative reserved storage: p.reserved=34359738368, reserved: -16384

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#7375] [Storage Provider] [answered]
> 2021-09-11T18:01:22.169+0500    WARN    stores  stores/local.go:126     negative reserved storage: p.reserved=34359738368, reserved: -16384

A very common error, the known cause is: no space left on sealing SSD. If you find other reasons you can add them below

**Comment by stuberman** [comment]
I get this whenever AP is running on the worker, there is plenty of space left (8TB available) on NVMe It is a code problem.

> 2021-09-22T07:05:44.467Z	WARN	stores	stores/local.go:131	negative reserved storage: p.reserved=34359738368, reserved: -8192

**Reply by de-authority** [reply]
Yeah. so i think this err wasn't caused by only one reason.

**Reply by de-authority** [reply]
One of our community members  confirmed this reason.  slack link here:https://filecoinproject.slack.com/archives/CPFTWMY7N/p1631365762368300

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
possibly resolved in 1.11.3 https://github.com/filecoin-project/lotus/pull/7233
